services/explorer/.eslintrc.js

```diff
@@ -9,6 +9,20 @@ module.exports = {
       },
       rules: {
         'no-restricted-exports': 'off',
+        'prettier/prettier': [

```

Comment:
I am pretty sure this will break my code again....

Reply:
It may do so, but this is a place it was missing, we should have it here too. Just reformat :D